components/global_header/left_controls/product_menu/product_menu_list/index.ts

```diff
@@ -45,6 +52,13 @@ function mapStateToProps(state: GlobalState) {
     const showVisitSystemConsoleTour = step === TaskNameMapToSteps[OnboardingTasksName.VISIT_SYSTEM_CONSOLE].STARTED;
     const enableCustomUserGroups = isCustomGroupsEnabled(state);
 
+    const subscription = getCloudSubscription(state);
+    const license = getLicense(state);
+
+    const isCloud = isCloudLicense(license);
+    const isCloudFreeEnabled = cloudFreeEnabled(state);
```

Comment:
We can also drop this feature flag now, we will be removing it across the codebase soon

Reply:
Added tests.
Removed FF.